examples/api/lib/painting/axis_direction/axis_direction.0.dart

```diff
@@ -0,0 +1,208 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// Flutter code sample for [AxisDirection]s.
+
+import 'package:flutter/material.dart';
+
+void main() => runApp(const ExampleApp());
+
+class ExampleApp extends StatelessWidget {
+  const ExampleApp({super.key});
+
+  static const String _title = 'Flutter Code Sample';
+
+  @override
+  Widget build(BuildContext context) {
+    return const MaterialApp(
+      title: _title,
+      home: MyWidget(),
+    );
+  }
+}
+
+class MyWidget extends StatefulWidget {
+  const MyWidget({ super.key });
+
+  @override
+  State<MyWidget> createState() => _MyWidgetState();
+}
+
+class _MyWidgetState extends State<MyWidget> {
+  final List<String> alphabet = <String>[
+    'A', 'B', 'C', 'D', 'E', 'F', 'G', 'H', 'I', 'J', 'K', 'L', 'M', 'N', 'O',
+    'P', 'Q', 'R', 'S', 'T', 'U', 'V', 'W', 'X', 'Y', 'Z',
+  ];
+  final Widget spacer = const SizedBox.square(dimension: 10);
+  Axis axis = Axis.vertical;
+  bool reverse = false;
```

Comment:
nit: these should probably be private?

Reply:
Oh I am actually curious about this. I feel like I came across a lint similar to this case. If the State object is private, do these need to be private?

Reply:
The rule I always use: If I were to make this object public, would we want people to access these properties? If no, then make them private.

For as long as the object stays private, it doesn't really matter. However, the current situation is inconsistent: These properties are public, but the methods below (_getAxisDirection etc) are private.